plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -71,10 +71,18 @@ export const OwnershipCard = (props: {
     hideRelationsToggle === undefined ? false : hideRelationsToggle;
   const classes = useStyles();
   const { entity } = useEntity();
-  const [getRelationsType, setRelationsType] = useState(
-    relationsType || 'direct',
+
+  const defaultRelationsType = entity.kind === 'User' ? 'aggregated' : 'direct';
+  const [derivedRelationsType, setRelationsType] = useState(
+    relationsType ?? defaultRelationsType,
   );
 
+  useEffect(() => {
+    if (!relationsType) {
+      setRelationsType(defaultRelationsType);
+    }
+  }, [setRelationsType, entity, defaultRelationsType, relationsType]);
```

Comment:
Isn't `relationsType` and `setRelationsType` enough for this useEffect? 🤔

Reply:
Yea you are right, the `entity` dep is not required actually, although `defaultRelationsType` seemingly is.